src/components/manage/Sharing/Sharing.jsx

```diff
@@ -83,6 +83,10 @@ const messages = defineMessages({
     id: 'Permissions have been updated successfully',
     defaultMessage: 'Permissions have been updated successfully',
   },
+  assignNewRoles: {
+    id: 'Assign the {entry} to the {role} role',
+    defaultMessage: 'Assign the {entry} to the {role} role',
```

Comment:
This seems backwards in English; the checkbox assigns a role to a user or group. I think it should be:

```suggestion
    id: 'Assign the {role} role to {entry}',
    defaultMessage: 'Assign the {role} role to {entry}',
```

Reply:
Thanks @davisagli. I made the changes you suggested.

Reply:
@Wagner3UB It's better, but in English we would say "Assign the Editor role to davisagli", not "Assign the Editor role to the davisagli"